public/docs/ts/latest/guide/security.jade

```diff
@@ -103,15 +106,17 @@ h2#xss Preventing cross-site scripting (XSS)
   a value that an attacker might control into `innerHTML` normally causes an XSS
   vulnerability. For example, code contained in a `<script>` tag is executed:
 
-+makeExcerpt('app/inner-html-binding.component.ts ()', 'inner-html-controller')
++makeExcerpt('app/inner-html-binding.component.ts', 'class')
 
-:marked
-  Angular recognizes the value as unsafe and automatically sanitizes it, which removes the `<script>`
-  tag but keeps safe content such as the text content of the `<script>` tag, or the `<b>` element.
+block html-sanitization
+  :marked
+    Angular recognizes the value as unsafe and automatically sanitizes it, which removes the `<script>`
+    tag but keeps safe content such as the text content of the `<script>` tag, or the `<b>` element.
+
+  figure.image-display
+      img(src='/resources/images/devguide/security/binding-inner-html.png'
+          alt='A screenshot showing interpolated and bound HTML values')
 
-figure.image-display
-    img(src='/resources/images/devguide/security/binding-inner-html.png'
-        alt='A screenshot showing interpolated and bound HTML values')
 :marked
   ### Avoid direct use of the DOM APIs
```

Comment:
This could be dartier. Also, the dart:html APIs are safer than the raw DOM APIs (e.g., see [Element.innerHtml Sanitization](https://groups.google.com/a/dartlang.org/forum/#!topic/misc/kQD2-_hY_DQ)). Not sure how that affects the general message, though, that they should avoid directly changing the DOM.